### PR TITLE
feat(bootstrap): Change buttons in top pannel 

### PR DESCRIPTION
**Pull request by labdala**
feat(bootstrap): Change buttons in top pannel 

Change top left pannel buttons framework from Material UI to Bootstrap. 
The buttons included in this change are: 
 - background color,
 - view planes, 
 - change axes,
 - reset camera,
 - screenshot,
 - fullscreen, 
 - background color.
These changes follow the changes for the annotations button introduced in commits  e5fd827 and 08aaa09 .

Co-authored-by: Paul Elliott <paul.elliott@kitware.com>

### CODE REVIEW

File: src/Main/ScreenshotButton.jsx [hunk 1]
```diff
@@ -1,38 +1,34 @@
...
+    <OverlayTrigger
+      transition={false}
+      overlay={<Tooltip>Screenshot</Tooltip>}
     >
-      <IconButton
-        size="small"
+      <Button
+        className={cn('icon-button', {
+          checked:state.context.main.screenshotEnabled
+        })}
```
Comment (PaulHax) [inline src/Main/ScreenshotButton.jsx:22]: This bit of state `state.context.main.screenshotEnabled` does not toggle, but probably should gray out and disable the button when `screenshotEnabled === false`

Can then remove the `import cn from 'classnames'`

File: src/Main/ViewModeButtons.jsx [hunk 2]
```diff
@@ -1,136 +1,78 @@
...
+  const listItems = buttonList.map(({ title, dataName, imageSrc }) => (
+    <ViewButton
+      key={title}
+      title={title}
+      dataName={dataName}
+      imageSrc={imageSrc}
+      service={service}
+    ></ViewButton>
+  ))
+  return listItems
```
Comment (PaulHax) [inline src/Main/ViewModeButtons.jsx:75]: A masterpiece!  

My style would be to skip the variable names, like so: 
```
return [ { stuff }, { stuff } ].map((stuff ) => (<Component stuff>))
```

But you do you.

File: src/Main/ResetCameraButton.jsx [hunk 3]
```diff
@@ -1,40 +1,32 @@
...
-  )
+    <OverlayTrigger
+      transition={false}
+      overlay={<Tooltip>Reset camera [r]</Tooltip>}
+     >
+     <Button
+        className={cn('icon-button', {
+          checked:state.context.main.resetCameraEnabled
+        })}
+        onClick={() =>send('RESET_CAMERA')}
```
Comment (PaulHax) [inline src/Main/ResetCameraButton.jsx:23]: That spacing on `=>send`

**Review by PaulHax — APPROVED** [review]
Exclente!  The rest of the controls will drop in a blink!

File: src/Main/ViewModeButtons.jsx [hunk 4]
```diff
@@ -1,136 +1,78 @@
...
-          anchorEl: zPlaneButton.current,
-          disablePortal: true,
-          keepMounted: true
+    <OverlayTrigger
+      transition={false}
+      overlay={<Tooltip> {props.title} </Tooltip>}
+    >
+      <Button
+        className={cn('icon-button', {
+          checked: state.context.main[enableButton]
```
Comment (PaulHax) [inline src/Main/ViewModeButtons.jsx:28]: try this `checked: state.context.main.viewMode === props.dataName`

**Review by PaulHax — APPROVED** [review]

File: src/Main/ViewPlanesToggle.jsx [hunk 5]
```diff
@@ -47,27 +46,17 @@ function ViewPlanesToggle(props) {
...
-      <ToggleButton
-        size="small"
-        className="toggleButton"
-        value="visiblePlanes"
-        selected={planesVisible()}
-        onChange={handleToggle}
+    <OverlayTrigger transition={false} overlay={<Tooltip>View planes</Tooltip>}>
+      <Button
+        className={cn('icon-button', {
+          checked: state.context.main.viewPlanesEnabled
```
[the remark this answers is not in the document]
Comment (labdala) [reply src/Main/ViewPlanesToggle.jsx:52]: Nice catch! By "Lets remove that related code from here for now", do you mean convert it back to MUI?

[the remark this answers is not in the document]
Comment (PaulHax) [reply src/Main/ViewPlanesToggle.jsx:52]: Let me clarify: Remove the `checked: state.context.main.viewPlanesEnabled` and `cn` code.  Just set the className as a constant `className = 'icon-button'`.  Can then remove the `useActor` above and just `service.send(EVENT)`

[the remark this answers is not in the document]
Comment (labdala) [reply src/Main/ViewPlanesToggle.jsx:52]: Got it. Thank you so much!

File: src/Main/ViewModeButtons.jsx (hunk 2 above)
Comment (labdala) [reply src/Main/ViewModeButtons.jsx:75]: Wow, thanks Paul!

**Pull request merged**